events/triggers/install-screen

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+UMBREL_ROOT="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/../..)"
+
+check_root () {
+  if [[ $UID != 0 ]]; then
+    echo "This script must be run as root"
+    exit 1
+  fi
+}
+
+check_umbrel_os () {
+  [[ -f "/etc/default/umbrel" ]] && source "/etc/default/umbrel"
+  if [[ -z "${UMBREL_OS:-}" ]]; then
+    echo "This script must only be run on Umbrel OS"
+    exit 1
+  fi
+}
+
+check_root
+check_umbrel_os
+
+# This script installs a screen driver, then reboots Umbrel
+echo "dtoverlay=piscreen,speed=16000000,rotate=270" >> /boot/config.txt
```

Comment:
Is it worth doing something like the following in case this script can be run multiple times?
```
add_boot_option () {
   if ! grep -q "^${1}" /boot/config.txt; then
     echo ${1} >> /boot/config.txt
   fi
}

add_boot_option "dtoverlay=piscreen,speed=16000000,rotate=270"
add_boot_option "dtparam=spi=on"
```

Reply:
Thanks, I'll add that.